tutorial/deploy_jimm_microk8s.rst

```diff
@@ -237,8 +237,13 @@ Run the following commands:
     # A private and public key for macaroon based authentication with Juju controllers.
     juju config jimm public-key="<public-key>"
     juju config jimm private-key="<private-key>"
-    # A final redirect URL for login, normally set to the URL of the Juju dashboard.
-    juju config jimm final-redirect-url="http://test-jimm.localhost/auth/whoami"
+
+Optionally, if you have deployed Juju Dashboard, you can configure JIMM to enable browser flow for authentication:
```

Comment:
Let's tweak this slightly for the purposes of the tutorial so that the dashboard location is still configured so that the user can still test the browser flow. What do you think?

Reply:
Yeah, sure. I'll set it with the same `/auth/whoami` we had, but will point out that this should be replaced with dashboard URL.